templates/form/radios.html.twig

```diff
@@ -18,8 +18,9 @@
 <div{{ attributes.setAttribute("role", "radiogroup").addClass(classes) }}>
   {%- if _label is not empty %}
     <div
-      class="ecl-form-label{{ hide_label ? ' ecl-form-label--hidden'}}"
+      class="ecl-form-label"
       id="{{ _label_id }}"
+      {{ hide_label ? 'hidden'}}
```

Comment:
If not set this will produce an empty line in the markup, maybe we put this one on the line above?

Reply:
done